docs/design/submarine-server/architecture.md

```diff
@@ -0,0 +1,126 @@
+<!--
+  Licensed to the Apache Software Foundation (ASF) under one
+  or more contributor license agreements.  See the NOTICE file
+  distributed with this work for additional information
+  regarding copyright ownership.  The ASF licenses this file
+  to you under the Apache License, Version 2.0 (the
+  "License"); you may not use this file except in compliance
+  with the License.  You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+  Unless required by applicable law or agreed to in writing,
+  software distributed under the License is distributed on an
+  "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+  KIND, either express or implied.  See the License for the
+  specific language governing permissions and limitations
+  under the License.
+  -->
+## Motivation
+Due to the single responsibility of considering different components, and nowadays the core module depends the hadoop libs which can be setted optional.
```

Comment:
This is a little confusing to me. Could you please rephrase it? The motivation in my mind for the submarine server is to hide the underlying YARN/k8s so that submarine-server can act as a job server and manage all jobs submitted.

Reply:
Yes, I forgot to update it. I will update it later.